src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -64,4 +69,15 @@ class Repository(
   fun lookupAgency(id: String): AgencyLocation = agencyLocationRepository.findByIdOrNull(id)!!
   fun delete(offender: Offender) = offenderRepository.deleteById(offender.id)
   fun delete(people: Collection<Person>) = personRepository.deleteAllById(people.map { it.id })
+
+  fun lookupVisit(visitId: Long?): Visit {
```

Comment:
I'm guessing the parameter doesn't need to be optional here (causing the !! below)

Same for changeVisitStatus?

Reply:
I just removed the need for it in the test code, e.g. responsebody.visitId is nullable